workbench/src/renderer/server_requests.js

```diff
@@ -82,7 +82,6 @@ export function fetchDatastackFromFile(payload) {
       headers: { 'Content-Type': 'application/json' },
     })
       .then((response) => response.json())
-      .catch((error) => logger.error(error.stack))

```

Comment:
Just making sure I understand - this error would now be caught farther up?

Reply:
That's right. It felt better to add the user-facing alert message to UI component, so that's where the error-handling is now.